pyproject.toml

```diff
@@ -42,15 +42,14 @@ dependencies = [
     "jupyter_core",
     "jupyter-lsp>=1.5.1",
     "jupyter_server>=2.0.1,<3",
-    "jupyter_ydoc==0.3.0",
-    "jupyter_server_ydoc>=0.6.0,<0.7.0",
     "jupyterlab_server>=2.19.0,<3",
+    "jupyter_ydoc>=0.3.0,<0.4.0",
```

Comment:
> Just checking if there was any reason for keeping the strict == for jupyter_ydoc before? Probably because it was still early and likely to break often?

Not really. It was to ping it to v0.3.x. I change it to make it coherent with the other dependencies.

Reply:
:+1: